api/src/main/java/org/openmrs/Allergies.java

```diff
@@ -61,7 +61,7 @@ public void clear() {
 		allergies.clear();
 	}
 	
-	public void confirmNoKnownAllergies() throws APIException {
+	public void confirmNoKnownAllergies() throws RuntimeException {
```

Comment:
@manuelkollus Are you sure this is what was required? According to me, Its quiet superfluous to add a `throws` clause of a ` RuntimeException` since as a matter of fact its an unchecked `Exception`.

Reply:
You can actually argue about whether it's redundant or not but I would actually do it so that RuntimeException stays there instead of APIException but this still remains your decision as to what you implement in this project.